app/templates/help.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %}
+
+{% block title %}About{% endblock %}
+
+{% block content %}
+
+<div class="help__container">
+<h2>The Polar Federated Search provides the ability to create your own queries through the search text interface.</h2>
```

Comment:
I think put a line above this: `<h2>Advanced Text Search</h2>`

and convert this line to an `<h3>` will help people understand a little better what they can do.